collect_app/src/androidTest/java/org/odk/collect/android/feature/settings/MainMenuTest.java

```diff
@@ -36,8 +36,7 @@ public void configureWithQRCanBeDisabled() {
                 .clickOnString(R.string.qr_code)
                 .pressBack(new AdminSettingsPage(rule))
                 .pressBack(new MainMenuPage(rule))
-                .clickOnMenu()
-                .assertTextDoesNotExist(R.string.configure_via_qr_code);
+                .assertOverflowMenuDoesNotExist();
```

Comment:
We need it to keep `Configure via QR code` option. In this test the option is disabled in admin settings so that's why I assert it's not visible at all. I could remove the three dot's menu and display an icon for that option but it would be removed later either way so it doesn't make sense.

Reply:
I was thinking we'd just get rid of that feature now seeing as `Configure via QR code` is still available via "Admin Settings". As you say though the test is really about the ability to disable the feature so, we'd also need to remove that config. Let's merge this and discuss.